data/fixture_data/dummy_file_stat_tracker.csv

```diff
@@ -0,0 +1,5 @@
+game_id,team_id,HoA,result,settled_in,head_coach,goals,shots,tackles,pim,powerPlayOpportunities,powerPlayGoals,faceOffWinPercentage,giveaways,takeaways
```

Comment:
We can remove this file until we have more of a grasp around mock testing

Reply:
Deleted the file in question